prdoc/pr_7266.prdoc

```diff
@@ -0,0 +1,36 @@
+title: Add `offchain_localStorageClear` RPC method
+doc:
```

Comment:
Can you clean this prdoc a bit with a proper description? I don't think we need to have the checklist in here.